src/targets/gpu/include/migraphx/gpu/miopen.hpp

```diff
@@ -225,6 +225,15 @@ inline pooling_descriptor make_pooling(const migraphx::op::pooling& op)
         ss << op.mode;
         MIGRAPHX_THROW(ss.str());
     }
+    if(not std::all_of(
+           op.dilations.cbegin(), op.dilations.cend(), [](std::size_t d) { return d == 1; }))
+    {
+        std::stringstream ss("Unsupported dilations for pooling: [");
+        for(const auto& d : op.dilations)
+            ss << d << ",";
+        ss << "]";
+        MIGRAPHX_THROW(ss.str());
```

Comment:
Just do `MIGRAPHX_THROW("Unsupported dilations for pooling: [" + to_string_range(op.dilations) + "]")` instead.